README.md

```diff
@@ -53,9 +53,12 @@ To get started you will first need to satisfy the following pre-requisites:
 ## Pre-requisites
 1. An OpenShift cluster that is 4.11 or higher.
 2. You will need to be logged into this cluster as [cluster admin] via [oc client].
-3. The OpenShift Cluster must have OpenShift Pipelines 1.8 or higher installed. We recommend channel pipelines-1.8
-   on OCP 4.10 and pipelines-1.9 or pipelines-1.10 for OCP 4.11, 4.12 and 4.13.
-   Instructions [here][OCP Pipelines Operator].
+3. Based on which DSP version to install you will need to do the following:
+   1. For DSPv1: The OpenShift Cluster must have OpenShift Pipelines 1.8 or higher installed. We recommend channel pipelines-1.8
+      on OCP 4.10 and pipelines-1.9 or pipelines-1.10 for OCP 4.11, 4.12 and 4.13.
+   2. For DSPv2: The OpenShift Cluster must be Argo Workflows installed. You can either install it by following the instructions
+      [here](https://argo-workflows.readthedocs.io/en/latest/quick-start/).
```

Comment:
i'd argue defaulting to standard Argo installs may still be problematic.  Since this is the DSPO repo, i'd say lets only let users follow OUR standard procedure of using `make`

Reply:
I believe @hbelmiro 's tested it, but sure I can take it out.